vllm/multimodal/inputs.py

```diff
@@ -13,13 +15,14 @@
 import torch
 import torch.types
 from PIL.Image import Image
```

Comment:
we should probably lazy import this as well

Reply:
Do you know how to deal with its usage here?

https://github.com/vllm-project/vllm/blob/f6518b2b487724b3aa20c8b8224faba5622c4e44/vllm/multimodal/inputs.py#L27

Reply:
You can gated it in TYPE_CHECKING

Reply:
You can apply this diff file

[diff.patch](https://github.com/user-attachments/files/20180739/diff.patch)

Reply:
can you also rename the title to "reduce vLLM's import time for `vllm/multimodals`" thanks

Reply:
ah, I didn't know `TYPE_CHECKING` works for `TypeAlias`es too. Thanks

updated the title